test/regressions/TestViewer.js

```diff
@@ -49,6 +49,12 @@ function TestViewer(props) {
     };
   }, []);
 
+  const viewerBoxSx = {
+    display: 'block',
+    p: 1,
+    position: 'relative',
```

Comment:
Can we remove this one?

```suggestion
```

I think we want to be able to catch cases where demos relies on a special parent position properly to be set.

Reply:
I'll open a PR for this cc @Janpot

Reply:
https://github.com/mui/material-ui/pull/43743